examples/trpo_gym_tf_cartpole.py

```diff
@@ -34,9 +33,4 @@
     # optimizer=ConjugateGradientOptimizer(hvp_approach=FiniteDifferenceHvp(base_eps=1e-5))
 )
 
-run_experiment_lite(
-    algo.train(),
-    n_parallel=1,
-    snapshot_mode="last",
-    seed=1
-)
+run_experiment_lite(algo.train(), n_parallel=1, snapshot_mode="last", seed=1)
```

Comment:
This is ugly but probably what was demanded by YAPF. It's definitely better to have the list style.

You can disable YAPF when it is wrong using this comment.
```python
run_experiment_lite(
    algo.train(),
    n_parallel=1,
    snapshot_mode="last",
    seed=1
) # yapf: disable
```